frontend/src/i18n/locales/en/frequent.json

```
@@ -1,31 +1,32 @@
 {
-  "Add": "Add",
-  "Cancel": "Cancel",
-  "Delete": "Delete",
-  "Description": "Description",
-  "Disabled": "Disabled",
-  "Done": "Done",
-  "Edit": "Edit",
-  "Enabled": "Enabled",
-  "Extra Files": "Extra Files",
-  "Groups": "Groups",
-  "Loading...": "Loading...",
-  "Main": "Main",
-  "Name": "Name",
-  "No": "No",
-  "No instances.": "No instances.",
-  "None": "None",
-  "Package": "Package",
-  "Product ID": "Product ID",
-  "Required": "Required",
-  "Save": "Save",
-  "Search": "Search",
-  "Select": "Select",
-  "Unknown": "Unknown",
-  "Update": "Update",
-  "Updates": "Updates",
-  "Yes": "Yes",
-  "add": "add",
-  "next page": "next page",
-  "previous page": "previous page"
+  "add": "Add",
+  "cancel": "Cancel",
+  "delete": "Delete",
+  "description": "Description",
+  "disabled": "Disabled",
+  "done": "Done",
+  "edit": "Edit",
+  "enabled": "Enabled",
+  "extra_files": "Extra Files",
+  "groups": "Groups",
+  "loading": "Loading...",
```

Same about ellipsis.
```suggestion
  "loading": "Loading…",
```